pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -37,8 +40,13 @@ type NodeAffinity struct {
 	handle              framework.Handle
 	addedNodeSelector   *nodeaffinity.NodeSelector
 	addedPrefSchedTerms *nodeaffinity.PreferredSchedulingTerms
+
+	// logger is only meant to be used by background activities which don't
+	// have some other logger in their parent callstack.
+	logger klog.Logger
 }
 
+var _ framework.PreEnqueuePlugin = &NodeAffinity{}
```

Comment:
Hi @sanposhiho, I'm not sure if the nodeaffinity plugin should implement the `PreEnqueuePlugin` interface. IMO, It's not needed. Can you help me to confirm it because I'm not familiar with the scheduler framework.

Reply:
Yep, no need to have PreEnqueuePlugin.

Reply:
thx